vendor/vendor.json

```diff
@@ -24,6 +24,12 @@
 			"revision": "97646858c46433e4afb3432ad28c12e968efa298",
 			"revisionTime": "2017-08-22T15:24:03Z"
 		},
+		{
+			"checksumSHA1": "c0Z2sKLKi+IKRVzq0IzNvqvfCrQ=",
```

Comment:
Gentle reminder that all new dependencies need to be de-vendorized on Debian and Fedora. Is a sufficiently new version of this package available there? Is it well-maintained or do we need to add it to our maintenance list?

Reply:
Sorry, I don't understand what's needed to "de-vendorized"?

As for the library, I'm not familiar with evaluating golang resources and was following this suggestion: https://github.com/snapcore/snapd/pull/7490#discussion_r326161053

There's not a lot of functionality used here, so I could "roll my own".

Reply:
For the record, there's no need to vendor anything. The shlex package is part of snapd tree now under `github.com/snapcore/snapd/strutil/shlex`. The upstream was unmaintained, so we've adopted it a fixed some issues.

Reply:
@bboozzoo nice. Thanks!

Reply:
Thanks, I totally forgot about this.